corehq/apps/users/templates/users/email/deprecated_user.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n %}
+
+<p>{{ greeting }}</p>
+
+{% blocktrans %}
+  <p>
+    Your CommCare user {{ old_username }} has now been migrated to the new user:
+    {{ new_username }}.
+
+    The following data was copied to your new user:
+    <ul>
+      <li>General user data (name, phone number, email, etc.)</li>
+      <li>Project space memberships (including roles and locations)</li>
+      <li>Saved reports</li>
+      <li>Scheduled reports</li>
+    </ul>
+
+    The following data was not copied to your new user:
+    <ul>
+      <li>API keys</li>
+      <li>Pending CommCare invitations</li>
+    </ul>
+
+    The {{ old_username }} user is no longer active. You can log into your new user with the username
+    {{ new_username }}. If you are exempt from SSO, you can follow the <i>Forgot Password</i> workflow to successfully
```

Comment:
[This comment](https://github.com/dimagi/commcare-hq/pull/30655#discussion_r759404415) (from original PR) said that this "SSO" language had been scrapped. Was this missed?

Reply:
interesting, looks like it was never pushed to the branch from that original PR